### DISCUSSION TITLE
AlbumArtist tag question (bug ?)

### DISCUSSION THREAD

**Discussion by Emilot** [jcorporation/myMPD#1307] [Q&A] [answered]
myMPD version: [e.g. 8.1.0]

If i put "Various Artists" as tag to AlbumArtist tag, myMPD showing it to me in different "views".

if i rename it to "Various", everything is good.

**Expected behavior**

To behave the same way as "Various". A lot of music albums are tagged as "Various Artists" and not "Various".
It shouldn't matter of the name.

I can provide screenshots if needed.

This is a MPD issue or myMPD or format issue (like .wav) ?

**Comment by jcorporation** [comment]
Various has no special meaning. It is a tag value as all other tags. Grouping albums works by querying the mpd database grouped by albumartist tag and date. As an alternative you can use the MusicBrainz albumid to group albums. This is the preferred way and myMPD falls only back to the albumartist tag if no MusicBrainz tag was found.

**Reply by Emilot** [reply]
Then it is an myMPD bug. It should display it right, independently of the tag name, as you said.

If i give the "Various Artist" name, it showing wrong. If i rename it to "Various", it showing right.

**Reply by jcorporation** [reply]
I do not know what you mean. Please add a screenshot of the wrong grouping and the tags of the this songs.

**Reply by Emilot** [reply]
![tags2](https://github.com/jcorporation/myMPD/assets/18086896/9c1ef1ac-2014-42aa-a31f-0e200805ddeb)
![tags1](https://github.com/jcorporation/myMPD/assets/18086896/baeb2d17-efb1-48ec-951c-01e28d0efb0e)

If you see in the myMPD picture i have an album "Natural High 4". This album had the same AlbumArtist tag "Various Artists" and it was showing in multiple windows. After i rename it to "Various", it is showing right.

**Reply by jcorporation** [reply]
I doubt that this is the root cause. What tags are displayed in the view of the screenshot?

I use in my setup for some albums "Various" and for others "Various Artists" and for others "Soundtrack" and it works flawlessly.

You updated the MPD database afterwards and build the myMPD cache new?

**Reply by Emilot** [reply]
Do you use these tags with .aif files? I do not know if this is happening to .flac, etc... 

I updated MPD afterwards of course. myMPD cache cleared and Force update now, nothing changed.

**Reply by Emilot** [reply]
I have "Various Artists" too but in .flac files and works flawlessly, as you said. That is why it looks "weird" as an issue.

**Reply by jcorporation** [reply]
How does the albumartist tag looks in the album view and how the song details of such a songs look like?

**Reply by Emilot** [reply]
> Various has no special meaning. It is a tag value as all other tags. Grouping albums works by querying the mpd database grouped by albumartist tag and date. As an alternative you can use the MusicBrainz albumid to group albums. This is the preferred way and myMPD falls only back to the albumartist tag if no MusicBrainz tag was found.

Do you have a guide for this? I have never used MusicBrainz.

I just saw your script "tagart" and needs fanart.tv registration and key. As i see it has some fee to create a key, right?

**Reply by Emilot** [reply]
> How does the albumartist tag looks in the album view and how the song details of such a songs look like?

![tag3](https://github.com/jcorporation/myMPD/assets/18086896/20eb6fb7-c9bd-4b63-ae8f-523c19127311)

![tag4](https://github.com/jcorporation/myMPD/assets/18086896/6c43914d-90f7-4d6e-861a-0154355ff872)

**Reply by jcorporation** [reply]
There is no AlbumArtist for this song in this case. Either MPD does not read/emit it or your tagging software is the problem.

Sure you enabled it in MPD and myMPD?

For the other things please open a separate discussion.

**Reply by Emilot** [reply, marked as answer]
> There is no AlbumArtist for this song in this case. Either MPD does not read/emit it or your tagging software is the problem.
> 
> For the other things please open a separate discussion.

Thank you JC. It's the software. I change it through Foobar and it shows it correctly. Damn

Sorry to waste your time.
